### ISSUE TITLE
Withdrawal issue

### ISSUE THREAD

**Issue by isaackielma** [mysteriumnetwork/node#4662]
**Describe the bug**
Tried to withdrawal my tokens today and balance went to zero but no transaction in app and nothing on polygon blockchain. Nothing in my Metamask wallet either.

As you can see from my screenshot I had 10+ MYST available to withdrawal so yesterday I clicked the withdrawal button and it said it when through, my balance went to zero but no transaction happened whatsoever.

**Screenshots**
<img width="1419" alt="Screenshot 2022-01-26 at 11 50 25" src="https://user-images.githubusercontent.com/84313603/151158344-d61c547e-96de-4c57-9f3f-7519292c6e53.png">

**Additional context**
Waiting on additional info.

**Comment by Guillembonet** [comment]
Hey it would be great if we could get some logs, you can use the report bug button on node ui to send them. We have been seeing this issue lately and will work on fixing it. Anyway, your funds are safe and will get to you in the next withdrawal you make.

**Comment by etherunit** [comment]
@isaackielma Could request logs from this user?

**Comment by iovestor** [comment]
I am encountering the same issue - balance went to zero but no transaction to my Withdrawal Address occurred.

I tried to use the 'Report issue' functionality in my Node UI but I faced another bug - when I press 'SEND' button I get 'could not create github issue: invalid character '<' looking for beginning of value (path="feedback/issue")'
I've reported another issue for the reporting issue:
https://github.com/mysteriumnetwork/node/issues/4663

**Comment by Guillembonet** [comment]
advise the user to retry a withdrawal and it will be fixed, I will create an issue so this problem doesn't happen again.
